data/index.css

```diff
@@ -1,56 +1,293 @@
+    font-size: 2rem;
+    margin-bottom: 20px;
+    color: #ffcc00;
+  }
+  
+  .about-section p {
+    font-size: 1rem;
+    max-width: 800px;
+    margin: 0 auto;
+  }
+  
+  /* Planets List */
+  .planets-list {
+    padding: 60px 20px;
+    background: rgba(0, 0, 0, 0.7);
+    text-align: center;
+  }
+  
+  .planets-list h2 {
+    font-size: 2rem;
+    margin-bottom: 20px;
+    color: #ffcc00;
+  }
+  
+  .planets-list ul {
+    list-style: none;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+  }
+  
+  .planets-list li {
+    margin: 10px 20px;
+  }
+  
+  .planets-list a {
+    color: #ffffff;
+    text-decoration: none;
+    font-size: 1.1rem;
+    padding: 10px 15px;
+    border: 2px solid #ffcc00;
+    border-radius: 5px;
+    transition: background 0.3s ease, color 0.3s ease;
+  }
+  
+  .planets-list a:hover {
+    background: #ffcc00;
+    color: #333333;
+  }
+  
+  /* Facts Section */
+  .facts-section {
+    padding: 60px 20px;
+    background: rgba(0, 0, 0, 0.8);
+    text-align: center;
+  }
+  
+  .facts-section h2 {
+    font-size: 2rem;
+    margin-bottom: 20px;
+    color: #ffcc00;
+  }
+  
+  #fact-container {
+    max-width: 800px;
+    margin: 0 auto;
+    padding: 20px;
+    background: rgba(255, 255, 255, 0.1);
+    border-radius: 10px;
+    position: relative;
+  }
+  
+  .fact-text {
+    font-size: 1.3rem;
+    line-height: 1.6;
+    color: #ffffff;
+    position: relative;
+  }
+  
+  .fact-text::before {
+    content: "✨";
+    position: absolute;
+    left: -30px;
+    top: 0;
+    font-size: 1.5rem;
+  }
+  
+  .fact-text::after {
+    content: "✨";
+    position: absolute;
+    right: -30px;
+    top: 0;
+    font-size: 1.5rem;
+  }
+  
+  /* Footer Styles */
+  footer {
+    background: rgba(0, 0, 0, 0.8);
+    text-align: center;
+    padding: 20px;
+    position: relative;
+  }
+  
+  footer p {
+    color: #ffffff;
+    font-size: 0.9rem;
+  }
+  
+  /* For Responsive Styles */
+  @media (max-width: 768px) {
+    .nav-links {
+      position: fixed;
+      right: 0;
+      height: 100vh;
+      top: 60px;
+      background: rgba(0, 0, 0, 0.9);
+      flex-direction: column;
+      align-items: center;
+      width: 200px;
+      transform: translateX(100%);
+      transition: transform 0.3s ease;
+    }
+  
+    .nav-links.active {
+      transform: translateX(0);
+    }
+  
+    .nav-links li {
+      margin: 20px 0;
+    }
+  
+    .hamburger {
+      display: block;
+    }
+  
+    .hero h1 {
+      font-size: 2.5rem;
+    }
+  
+    .hero p {
+      font-size: 1rem;
+    }
+  
+    .planets-list ul, .facts-section ul {
+      flex-direction: column;
+      align-items: center;
+    }
+  
+    .planets-list li, .facts-section li {
+      margin: 10px 0;
+    }
+  
+    #fact-container {
+      padding: 15px;
+    }
+  
+    .fact-text {
+      font-size: 1.1rem;
+    }
+  
+    .fact-text::before,
+    .fact-text::after {
+      display: none;
+    }
+  }
+  
```

Comment:
Indentation unusual.  Trailing space without end of line marker.  My terminal prompt shifted right.

```
$ cat index.css
    }
  }
  $ █
```